editor/javascript/configuration/apollo.js

```diff
@@ -0,0 +1,27 @@
+import {ApolloClient} from 'apollo-client';
+import {ApolloLink} from 'apollo-link';
+import {HttpLink} from 'apollo-link-http';
+import {InMemoryCache} from 'apollo-cache-inmemory';
+import {onError} from 'apollo-link-error';
+
+const Client = new ApolloClient({
+  link: ApolloLink.from([
+    onError(({graphQLErrors, networkError}) => {
+      if (graphQLErrors)
+        graphQLErrors.map(({message, locations, path}) =>
+          console.log(`[GraphQL error]:
```

Comment:
Personally I'd prefer to use an actual `console.error` here. Light up that console!